remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/CompatibilityProtoStreamMarshaller.java

```diff
@@ -3,7 +3,7 @@
 import org.infinispan.factories.annotations.Inject;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.protostream.SerializationContext;
-import org.infinispan.query.remote.client.BaseProtoStreamMarshaller;
```

Comment:
woud it be possible not to move BaseProtoStreamMarshaller to commons ?

Reply:
Yeah good point. I will move this and avoid adding the dependency to `infinispan-commons` as well. Originally I though that there would be more complex changes to `BaseProtoStreamMarshaller` when I thought implementing `StreamingMarshaller` was required.